timelock_auction/tlock_proxy/lib.rs

```diff
@@ -259,20 +248,20 @@ mod tlock_proxy {
             Ok(())
         }
 
+        // Reveals a single bid
         #[ink(message)]
-        pub fn get_encrypted_bids(
-            &self,
+        pub fn reveal_bid(
```

Comment:
reveal_bid and bid don't need to be two separate functions, this could just be a single call (invoked via a delayed transaction)